docs/modules/ROOT/pages/how-tos/aks_recover_from_backup.adoc

```diff
@@ -0,0 +1,141 @@
+export AZURE_ACCOUNT_NAME=$(kubectl get secret admin-credentials-minio-k8up -n syn-minio -o jsonpath='{.data.accesskey}' | base64 --decode)
+export AZURE_ACCOUNT_KEY=$(kubectl get secret admin-credentials-minio-k8up -n syn-minio -o jsonpath='{.data.secretkey}' | base64 --decode)
+----
+
+=== Obtaining restic configuration from catalog and vault
+
+. Obtain the repository URL of the cluster's catalog and export it to `REPO_URL`
++
+Get the URL from https://control.vshn.net/syn/lieutenantclusters.
+Alternatively, get it from the Lieutenant API or the Kubernetes API Lieutenant is running on.
++
+.Use the Lieutenant API
+[source,bash]
+----
+REPO_URL=$(curl -sH "${LIEUTENANT_AUTH}" "https://${LIEUTENANT_URL}/clusters/${CLUSTER_ID}" | jq -r .gitRepo.url)
+----
++
+.Use the Kubernetes API
+[source,bash]
+----
+REPO_URL=$(kubectl -n ${LIEUTENANT_NS} get cluster -o jsonpath='{.spec.gitRepoURL}' ${CLUSTER_ID})
+----
+
+. Download and extract the cluster catalog
++
+[source,bash]
+----
+mkdir catalog
+git archive --remote ${REPO_URL} master | tar -xC catalog
+----
+
+. Login to Vault
++
+[source,bash]
+----
+export VAULT_ADDR=https://vault-prod.syn.vshn.net
+vault login -method=ldap username=<your username>
+----
+
+. Export restic configuration
++
+[source,bash]
+----
+export RESTIC_REPOSITORY=azure:$(yq -r '. | select(.kind == "Schedule").spec.backend.s3.bucket' catalog/manifests/cluster-backup/05_schedule.yaml):/
+
+PASSWORD_KEY="$(yq -r '. | select(.kind == "Secret" and .metadata.name == "objects-backup-password").stringData.password' catalog/manifests/cluster-backup/05_schedule.yaml | cut -d: -f2)"
+export RESTIC_PASSWORD=$(vault kv get -format json "clusters/kv/${PASSWORD_KEY%/*}" | jq -r ".data.data.${PASSWORD_KEY##*/}")
+
+ID_KEY="$(yq -r '. | select(.kind == "Secret" and .metadata.name == "object-backup-s3-credentials").stringData.username' catalog/manifests/cluster-backup/05_schedule.yaml | cut -d: -f2)"
+export AZURE_ACCOUNT_NAME=$(vault kv get -format json "clusters/kv/${ID_KEY%/*}" | jq -r ".data.data.${ID_KEY##*/}")
+
+SECRET_KEY="$(yq -r '. | select(.kind == "Secret" and .metadata.name == "object-backup-s3-credentials").stringData.password' catalog/manifests/cluster-backup/05_schedule.yaml | cut -d: -f2)"
+export AZURE_ACCOUNT_KEY=$(vault kv get -format json "clusters/kv/${SECRET_KEY%/*}" | jq -r ".data.data.${SECRET_KEY##*/}")
+----
+
+== Identify and retrieve snapshot
+
+. List the available snapshots.
+   Identify the one you do want to restore.
+   Take note of its `ID`.
++
+[source,bash]
+----
+restic snapshots
+----
+
+. Retrieve the backup archive
++
+[source,bash]
+----
+restic restore <ID> --target cluster-backup-object-restore-$(date +%F)
+----
+
+== Extract and prepare files
+
+. List files in the backup.
+   Take note of the path containing the required files.
++
+[source,bash]
+----
+tar tvf syn-cluster-backup-object-dumper.tar.gz
```

Comment:
After checking the documentation for `restic`, this command needs to be executed in the directory created by restic in the previous step, since `--target` specifies the directory to which to extract the snapshot.

Reply:
Added the `cd`ahead to switch into the target directory.